plugins/finder/jevents/jevents.php

```diff
@@ -361,21 +305,35 @@ protected function getItem($id)
 			throw new Exception($this->db->getErrorMsg(), 500);
 		}
 
+		// query the repeat table to get start and end of repeats
+		$sql = $this->getRepeats();
```

Comment:
Since custom method getRepeats() got deleted, I think we can merge the query in getListQuery() method to get the repetition start and end date to populate publish_start_date and publish_end_date. 

I just separated them just because I don't have time to make the query clean since the query is one to one-to-many and it will make the getListQuery() sql statement more complicated.

If I'm not mistaken, I think publish_start_date and publish_end_date has to be populated so finder will auto remove event from the search result.